packages/aws-rfdk/package.json

```diff
@@ -47,19 +43,20 @@
     ]
   },
   "scripts": {
-    "awslint": "cdk-awslint",
-    "build": "cdk-build",
+    "awslint": "awslint",
+    "build": "jsii --project-references --silence-warnings=reserved-word && yarn run lint",
     "build+test": "yarn run build && yarn test",
     "build+test+package": "yarn run build+test && yarn run package",
     "clean": "tsc --build --clean && bash ./clean.sh",
-    "lint": "cdk-lint",
-    "lintfix": "cdk-lint --fix",
-    "package": "cdk-package",
-    "pkglint": "pkglint -f",
+    "lint": "eslint . --ext=.ts --resolve-plugins-relative-to=. && yarn run pkglint",
+    "lintfix": "eslint . --ext=.ts --resolve-plugins-relative-to=.",
```

Comment:
I don't see any difference between the the `eslint` portions of these commands. Is there an additional flag we should be passing to `eslint` to make it fix the source code?

Reply:
Fixed in revision. Thanks for spotting that!